be/src/olap/txn_manager.cpp

```diff
@@ -168,6 +168,32 @@ Status TxnManager::prepare_txn(TPartitionId partition_id, TTransactionId transac
     return Status::OK();
 }
 
+Status TxnManager::set_txn_related_delete_bitmap(TPartitionId partition_id,
+                                                 TTransactionId transaction_id, TTabletId tablet_id,
+                                                 SchemaHash schema_hash, TabletUid tablet_uid,
+                                                 bool unique_key_merge_on_write,
+                                                 DeleteBitmapPtr delete_bitmap,
+                                                 const RowsetIdUnorderedSet& rowset_ids) {
+    pair<int64_t, int64_t> key(partition_id, transaction_id);
+    TabletInfo tablet_info(tablet_id, schema_hash, tablet_uid);
+
+    std::unique_lock<std::mutex> txn_lock(_get_txn_lock(transaction_id));
+    {
+        // get tx
+        std::shared_lock rdlock(_get_txn_map_lock(transaction_id));
+        txn_tablet_map_t& txn_tablet_map = _get_txn_tablet_map(transaction_id);
+        auto it = txn_tablet_map.find(key);
+        DCHECK(it != txn_tablet_map.end());
+        auto load_itr = it->second.find(tablet_info);
+        DCHECK(load_itr != it->second.end());
+        TabletTxnInfo& load_info = load_itr->second;
+        load_info.unique_key_merge_on_write = unique_key_merge_on_write;
+        load_info.delete_bitmap = delete_bitmap;
+        load_info.rowset_ids = rowset_ids;
+    }
+    return Status::OK();
```

Comment:
the function always return OK, so it should be a void function.

Reply:
fixed